src/ucp/wireup/wireup_ep.c

```diff
@@ -479,6 +479,17 @@ static UCS_CLASS_CLEANUP_FUNC(ucp_wireup_ep_t)
         ucp_wireup_replay_pending_requests(ucp_ep, &tmp_pending_queue);
     }
 
+    if (self->super.is_owner && (self->super.uct_ep != NULL)) {
+        ucp_worker_discard_uct_ep(self->super.ucp_ep, self->super.uct_ep,
```

Comment:
1. fix indentation
2. can we unite common code with ucp_wireup_ep_discard_aux_ep() ?

Reply:
done